### PR TITLE
Add first version of Dockerfile and docker-compose

### PR DESCRIPTION
**Pull request by ldlamarc**
Enabling building and running from source

### CODE REVIEW

**Comment by ldlamarc** [comment]
Some optimizations are still possible: 

- copying dist folder to a smaller Docker image without build tools aka multi-stage build => end image is smaller
- using the output of this Dockerfile as image for the registry => more consistent and transparent
- source conf folder including routes and translations => more simplicity, we can mount the whole conf folder in docker-compose
- dockerignore file => less files in end or intermediary images
- reviewing the README regarding prebuilt docker image, conf folder should also be mounted there I think to enable editing

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+services:
+  app:
+    stdin_open: true
+    tty: true
+    image: quizmaster
+    ports:
+      - "9000:9000"
+    volumes:
+      - ./conf/quiz:/src/dist/server-1.0/conf/quiz
+      - ./conf/application.conf:/src/dist/server-1.0/conf/application.conf
```
Comment (nymanjens) [inline docker-compose.yml:11]: I don't think application.conf should ever be changed, so best remove this

File: README.md [hunk 2]
```diff
@@ -9,28 +8,41 @@ A web-app for conducting a quiz, including a page for players to enter their ans
...
 - Run the following command:
 
 ```
 docker run nymanjens/quizmaster:latest bin/server
 ```
 
-- Browse to `http://<ip_address>:9000`
+## Building and running your own release with Docker
+
+- Run the following command:
```
Comment (nymanjens) [inline README.md:30]: Prefix this with git checkout github/... ?

**Review by nymanjens — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (ldlamarc) [reply docker-compose.yml:11]: I had to change app.quiz.configYamlFilePath = .

File: README.md (hunk 2 above)
Comment (ldlamarc) [reply README.md:30]: yes checking out the project is a requirement here

**Pull request merged**